src/schema/input.ts

```diff
@@ -50,9 +50,9 @@ export const tmtArtifactsSchema = z
 // https://testing-farm.gitlab.io/api/#operation/requestsPost
 export const tmtContextSchema = z
   .object({
-    distro: z.string().min(1),
-    arch: z.string().min(1),
-    trigger: z.string().min(1),
+    distro: z.string().min(1).optional(),
```

Comment:
Don't we want to support all tmt context dimensions?
https://tmt.readthedocs.io/en/stable/spec/context.html#dimension